.github/workflows/bundle-size.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       - uses: actions/setup-node@v1
         with:
-          node-version: '14'
+          node-version: '18'
```

Comment:
It's intentional as Node 18 contains native fetch. fetch is required for that storage adapter.